### ISSUE TITLE
Switch Safe screeen animation is not as design - can and should we make it? (plus minor UI diff)

### ISSUE THREAD

**Issue by DmitryBespalov** [gnosis/safe-ios#67]
- [ ] When the screen is just moving up, the closing button "X" is not visible on the design - is this is how it needed?
- [ ] There is a horizontal mark in the middle of the top edge in the designs which disappears when the screen is full screen.
- [ ] The safe name should be further apart from the safe address in the cells

<img src="https://images.zenhubusercontent.com/5c361cfa7727940fef83693b/3bae810a-b3d4-43b1-95dc-710b8a18d2a2" width=600px/>

<img src="https://images.zenhubusercontent.com/5c361cfa7727940fef83693b/51b15084-0df0-4ba7-8b34-6581cf7f9ae9" width=600px/>

**Comment by DmitryBespalov** [comment]
Response from Sasha: the close button appears when the screen is presented fully.

**Comment by sche** [comment]
I talked to @posthnikova about this issue one more time.
Let's wait for a final resolution.

**Comment by posthnikova** [comment]
Let's not spend time on this bottom sheet now.

**Comment by tschubotz** [comment]
I agree as well. Let's not spend time on this. I added ad note in the epic #285 about this for reference.
